src/bulk-load.ts

```diff
@@ -508,6 +515,21 @@ class BulkLoad extends EventEmitter {
       addOptions.push('TABLOCK');
     }
 
+    if (this.bulkOptions.order.length) {
+      const orderColumns = [];
+
+      for (const column of this.bulkOptions.order) {
+        // Ensure column exists as a specified column, otherwise we ignore.
+        if (column.columnName && this.columnsByName[column.columnName] !== null) {
+          orderColumns.push(`${column.columnName} ${column.direction}`);
```

Comment:
Do we need to be more careful with the column names here, in regards to SQL injection problems?

Reply:
I'd imagine there exists a risk/scenario where a malicious columnName can do unintended actions, so I chose to (almost on a whim) ignore columnNames that didn't exist. But looking at the situation from Tedious's perspective, it might not be the package's responsibility? Not sure - I'll wait for your guidance here 🙏

Also if we do chose to ignore 'invalid' columnNames, it might cause some developer frustration if we ignore their commands without throwing some warning message that they might've mistyped something. This might be an area where we can save some human-hours. Thoughts?

Reply:
@arthurschreiber Hey, friendly ping for this PR 😄 any action items for me?